Source/GB.tnLabs.Web/API_Controllers/TrainingsController.cs

```diff
@@ -7,7 +7,7 @@
 using Newtonsoft.Json.Linq;
 using Breeze.WebApi2;
 
-namespace GB.tnLabs.Web.Controllers
+namespace GB.tnLabs.Web.API_Controllers
```

Comment:
the folder name and the namespace name should be APIControllers not API_Controllers. (Bad practice)